src/Illuminate/Http/Client/PendingRequest.php

```diff
@@ -162,6 +162,7 @@ public function __construct(Factory $factory = null)
 
         $this->options = [
             'http_errors' => false,
+            'timeout' => 10,
```

Comment:
10 seconds is probably too low for a lot of people. I'd recommend going with:

```
'connect_timeout' => 10,
'timeout' => 15,
```

like in https://github.com/GrahamCampbell/Guzzle-Factory/blob/5.0/src/GuzzleFactory.php.